api/src/main/java/com/codeforcommunity/dto/report/ReportGeneric.java

```diff
@@ -18,6 +19,7 @@
   protected String visitReason;
   protected String actionPlan;
   protected String successStories;
+  protected Grade[] gradesAttended;
```

Comment:
Instead of an array of Grades, I would make it a List<Grade> everywhere where Grade[] appears

Reply:
Yeah I wanted to do that but I didn't know how to do lists in the database so I just did arrays

Reply:
You can just change it to an array list and it will work as expected. Also, the migration file needs to be written to have this change appear in the DB

Reply:
Have you tested this @DerekL27 ? In the DB, we want this to be one column, and in the JSON requests / responses we want this to be a list of string